tests/0137-barrier_batch_consume.c

```diff
@@ -87,6 +87,8 @@ static int consumer_batch_queue(void *arg) {
                 rd_kafka_message_destroy(rkmessage[i]);
         }
 
+        rd_free(rkmessage);
```

Comment:
This function is taken from TEST 0122 as it is. We should add this in that test as well.